src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -196,6 +196,12 @@ public CreateContainerResponse exec() throws NotFoundException, ConflictExceptio
         return aliases;
     }
 
+    @Override
+    @JsonIgnore
+    public Boolean getAutoRemove() {
+      return hostConfig.getAutoRemove();
```

Comment:
don't proxy calls to hostconfig. If you need this value get hostconfig object and get value from it